Packs/WolkenITSM/Integrations/WolkenITSM/WolkenITSM.py

```diff
@@ -0,0 +1,607 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+'''IMPORTS'''
+import traceback
+from typing import Any, Dict
+
+import urllib3
+
+# from _collections import defaultdict
+
+
+class Client(BaseClient):
+    def __init__(self, server_url, verify, proxy, headers, auth, clientId, domain, serviceAccount, refresh_token):
+        super().__init__(base_url=server_url, verify=verify, proxy=proxy, headers=headers, auth=auth)
+
+    def add_internal_notes_request(self, incidentId, clientId, domain, serviceAccount, Authorization, formData):
+
+        formData = assign_params(formData=formData)
+        headers = self._headers
+        headers.update({
+            'Authorization': 'Bearer ' + Authorization
+        })
+
+        response = self._http_request('post', f'api/v1/incidents/{incidentId}/addInternalNotes', data=formData, headers=headers)
+        return response
+
+    def create_incident_requestv1_request(self, clientId, domain, serviceAccount, Authorization, formData, file):
+        formData = assign_params(formData=formData)
+        formData = str(formData)
+        headers = self._headers
+        headers.update({
+            'Authorization': 'Bearer ' + Authorization
+        })
+
+        headers['accept'] = 'application/json'
+        print(formData)
+        print(headers)
+        print(type(formData))
+        response = self._http_request('post', 'api/v1/incidents', data=formData, headers=headers)
+        return response
+
+    def get_access_token_request(self, Authorization, domain, grant_type, refresh_token):
+        """
+        Generate a refresh token using the given client credentials and save it in the integration context.
+        """
+        data = assign_params(grant_type=grant_type, refresh_token=refresh_token, domain=domain)
+        headers = self._headers
+        headers['accept'] = 'application/json'
+
+        try:
+            res = self._http_request('POST', 'oauth/token', data=data, headers=headers)
+            time_now = date_to_timestamp(datetime.now())
+
+            TokenParam = {
+                'domain': domain,
+                'grant_type': grant_type,
+                'Authorization': Authorization,
+                'refresh_token': res.get('refresh_token'),
+                'access_token': res.get('access_token'),
+                'expires_in': res.get('expires_in'),
+                'scope': res.get('scope'),
+                'token_type': res.get('token_type'),
+                'time_now1': time_now
+            }
+
+            set_integration_context(TokenParam)
```

Comment:
Because if any other command is run then TokenParam contains Bearer access token which will be used to call any other api endpoint in other commands.